src/aleph/chains/indexer_reader.py

```diff
@@ -146,8 +146,8 @@ async def _query(self, query: str, model: Type[T]) -> T:
 
         response = await self.http_session.post("/", json={"query": query})
         response.raise_for_status()
-        response_json = await response.json()
-        return model.parse_obj(response_json)
+        response_json = await response.model_dump_json()
```

Comment:
ClientResponse doesn't have .model_dump_json() as pydantic is not used by Aiohttp.